test/geometry/transform/test_affine.py

```diff
@@ -698,3 +698,23 @@ def test_compose_affine_matrix_3x3(self, device, dtype):
         )
         matrix_kornia = matrix_kornia.inverse()[0, :2].detach().cpu()
         assert_close(matrix_kornia, matrix_expected, atol=1e-4, rtol=1e-4)
+
+
+class TestGetAffineMatrix:
+    def test_smoke(self, device, dtype):
+        H, W = 5, 5
+        translation = torch.tensor([[0., 0.]], device=device, dtype=dtype)
+        # NOTE: ideally the center should be [W * 0.5, H * 0.5]
+        center = torch.tensor([[W // 2, H // 2]], device=device, dtype=dtype)
```

Comment:
@ducha-aiki check this

Reply:
LGTM

Reply:
well something weird is happening with cuda in the ci when we concatenate matrices - however in local works well as expected